app/src/main/java/org/go/sopt/SignUpActivity.kt

```diff
@@ -6,100 +6,79 @@ import android.os.Bundle
 import android.util.Log
 import android.view.inputmethod.InputMethodManager
 import androidx.appcompat.app.AppCompatActivity
+import androidx.lifecycle.ViewModel
 import com.example.seminar1.databinding.ActivitySignUpBinding
 import com.google.android.material.snackbar.Snackbar
 import org.go.sopt.model.RequestSignUpDto
 import org.go.sopt.model.ResponseSignUpDto
+import org.go.sopt.viewmodel.SignUpViewModel
 import retrofit2.Call
 import retrofit2.Response
 
 class SignUpActivity : AppCompatActivity() {
 
     private lateinit var binding: ActivitySignUpBinding
-    private val signUpService = ServicePool.signUpService
+    private lateinit var viewModel: SignUpViewModel
```

Comment:
```suggestion
    private val viweModel by viewModels<SignUpViewModel>()
```

이렇게 바꾸는 게 좋은 이유를 두 가지 정도 말씀드리자면, 
- by 를 통해 위임을 사용하면 액티비티 생명주기 내에서 viewModel이 자동으로 제거됩니다. 좀 더 안정적이죠?
- 또한 초기화를 자동으로 처리하므로 코드가 간결해집니다. 

야무지죠?